plone/app/textfield/browser.py

```diff
@@ -43,7 +42,7 @@ def __call__(self, value=None, fieldName=None, mimeType=None):
             if not self.major or not self.minor:
                 mimeType = value.outputMimeType
             else:
-                mimeType = "%s/%s" % (self.major, self.minor,)
+                mimeType = "{}/{}".format(self.major, self.minor)
```

Comment:
Running `pyupgrade --py38-plus` would turn this into a nice f-string 😄

Reply:
Well, this is what I did: `find . -name "*.py" -exec pyupgrade --py38-plus {} +`.
Anyway, running it again turned it into an f-string.

Reply:
I see that too: sometimes running pyupgrade twice still makes changes in the second run.

Reply:
Maybe it's `pyupgrade` + `black` + `pyupgrade` ? 🤔 oh, well, `pre-commit` will warn us 🍀